src/main/java/io/voucherify/client/model/product/response/SKUResponse.java

```diff
@@ -28,5 +28,14 @@
   @JsonProperty("created_at")
   private Date createdAt;
 
+  @JsonProperty("image_url")
+  private String imageUrl;
+
+  private String currency;
+
+  private String price;
```

Comment:
why `String`? it should be a number just like in ProductResponse

Reply:
Hi. I have updated the price param to Long. Thanks for the correction.